python/setup.py

```diff
@@ -0,0 +1,29 @@
+from setuptools import setup
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setup(
+  name = "do_username",
+  version = "1.0.2",
+  packages = ["do_username"],
+  license = "MIT",
+  author = "Shreyas",
+  author_email = "shreyas.sreenivasa@gmail.com",
+  url = "https://github.com/shreyas44/do_username_py",
+  description = "A Digital Ocean Community themed username generator",
```

Comment:
```suggestion
  description = "DigitalOcean Username Generator",
```
Consistent short description to match other packages